src/Accounts/Accounts/ChangeLog.md

```diff
@@ -19,7 +19,7 @@
 -->
 
 ## Upcoming Release
-* Supported returning all subscriptions with specified name while using `Get-AzSubscription` with paramater `SubscriptionName`. [#19295]
+* Supported returning all subscriptions with specified name while using `Get-AzSubscription` with parameter `SubscriptionName`. [#19295]
```

Comment:
cc @NoriZC I recommend installing spell checker extensions in your IDEs  for example [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)

Reply:
Sorry for this and thank you very much! I will install the extension soon.